### DISCUSSION TITLE
When will imx-manifest have a branch for Yocto Scarthgap?

### DISCUSSION THREAD

**Discussion by owzagj** [Freescale/meta-freescale#1808] [Q&A] [answered]
As yocto Scarthgap was announced for April 2024 there are still no imx-manifest to produce an image of it. Is there any plan to create it? If positive, when would it be done?

Thanks in advance!

**Comment by angolini** [comment, marked as answer]
Do you mean: https://github.com/Freescale/fsl-community-bsp-platform/tree/scarthgap?

If not, could you please elaborate?
